Formula/bareos-fd.rb

```diff
@@ -0,0 +1,70 @@
+class BareosFd < Formula
+  desc "Open-Source Data Protection Solution"
+  homepage "https://bareos.org"
+  url "https://github.com/bareos/bareos/archive/Release/16.2.7.tar.gz"
```

Comment:
Does this need updating to the latest release or is this still the stable release?

Reply:
The code for the next stable release is push to github today. It is planed to publish Linux/Windows (and a Mac PKG) packages on Monday.
So yes, since today this can be switched to 17.2.4.